examples/apps/likes-and-comments/package.json

```diff
@@ -33,12 +33,12 @@
   "dependencies": {
     "@fluid-experimental/get-container": "^0.39.0",
     "@fluidframework/aqueduct": "^0.39.0",
-    "@fluidframework/common-definitions": "^0.19.1",
+    "@fluidframework/common-definitions": "^0.20.0-0",
     "@fluidframework/counter": "^0.39.0",
     "@fluidframework/datastore-definitions": "^0.39.0",
     "@fluidframework/map": "^0.39.0",
     "@fluidframework/merge-tree": "^0.39.0",
-    "@fluidframework/protocol-definitions": "^0.1022.0",
+    "@fluidframework/protocol-definitions": "^0.1024.0-0",
```

Comment:
@tanviraumi are there any server changes we should be actively aware of when taking this newer version?  It seems a bit off that we managed to jump over a version here

Reply:
I don't recall any major changes in protocol. So it should be fine. We have also started to maintain a BREAKING.md for server (https://github.com/microsoft/FluidFramework/blob/main/server/BREAKING.md).